icu4c/source/i18n/unicode/calendar.h

```diff
@@ -1341,7 +1341,7 @@ class U_I18N_API Calendar : public UObject {
      *           same class ID. Objects of other classes have different class IDs.
      * @stable ICU 2.0
      */
-    virtual UClassID getDynamicClassID(void) const = 0;
+    //virtual UClassID getDynamicClassID(void) const = 0;
```

Comment:
This change the requirement for the subclass to implement the getDynamicClassID() and remove the compile/build time error if the subclass does not do so but push to run time assertion/error and I am concern about it.

Reply:
virtual UClassID getDynamicClassID(void) const override = 0;